appyx-navigation/common/src/commonMain/kotlin/com/bumble/appyx/navigation/node/Node.kt

```diff
@@ -33,25 +26,27 @@ import com.bumble.appyx.navigation.plugin.UpNavigationHandler
 import com.bumble.appyx.navigation.plugin.plugins
 import com.bumble.appyx.navigation.state.SavedStateMap
 import com.bumble.appyx.navigation.store.RetainedInstanceStore
+import com.bumble.appyx.utils.multiplatform.BuildFlags
+import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.withContext
 
 @Suppress("TooManyFunctions")
 @Stable
-open class Node @VisibleForTesting internal constructor(
+open class Node internal constructor(
     private val buildContext: BuildContext,
     val view: NodeView = EmptyNodeView,
     private val retainedInstanceStore: RetainedInstanceStore,
     plugins: List<Plugin> = emptyList()
-) : NodeLifecycle, NodeView by view, RequestCodeClient {
```

Comment:
Because request code is an Android concept that doesn't need to be entangled with a multiplatform Node. Unless I'm off-base here, android client code can supply any RequestCodeClient impl needed at the activity level, no?

Reply:
No, `RequestCodeClient` is needed on a per Node basis. We'll have to implement `RequestCodeClient` in the client code for Android Nodes

Reply:
in conversation with Zsolt about this there was a question as to whether we even needed this on Android for a good reason, so worth re-assessing perhaps?